common/djangoapps/student/tests/test_email.py

```diff
@@ -84,7 +85,7 @@ def append_allowed_hosts(self, hostname):
 
 
 @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
-class ActivationEmailTests(TestCase):
+class ActivationEmailTests(CacheIsolationTestCase):
```

Comment:
I *think* all of these were fixed in the CMS changes, might be worth trying them again without these changes

Reply:
Hm, the CMS changes did touch this file, but only to update module names and fix style.  The problem that we tried to fix here was where the `site` template context variable was bleeding across tests via the request cache.